ChangeLog.d/mpi_read_negative_zero.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Fix mbedtls_mpi_read_string on "-0" returning a ``negative zero'' object,
```

Comment:
I meant `"-0"` as in a C string and ``` ``negative zero'' ``` as English quotation marks. But I've rephrased to avoid using the expression “negative zero” altogether.